nvtabular/inference/triton/ensemble.py

```diff
@@ -30,14 +32,17 @@
 from nvtabular.dispatch import _is_list_dtype, _is_string_dtype  # noqa
 from nvtabular.graph.node import iter_nodes  # noqa
 from nvtabular.graph.schema import Schema  # noqa
+from nvtabular.graph.tags import Tags  # noqa
 
 
 def export_tensorflow_ensemble(
     model,
     workflow,
     name,
     model_path,
-    label_columns,
+    cats=None,
+    conts=None,
```

Comment:
The movielens notebook passes the label_column by position instead of by name

```"export_tensorflow_ensemble(model, workflow, MODEL_NAME_ENSEMBLE, MODEL_PATH, [\"rating\"])```

( https://github.com/NVIDIA-Merlin/NVTabular/blame/main/examples/getting-started-movielens/03-Training-with-TF.ipynb#L706 ).

I think this will still break that notebook, or at least cause unexpected results with the `rating` column being treated as a categorical.

Do we need to insert the cats/conts here?

Reply:
I just used the keyword arg in the call (label_columns) and now everything is working again 👍  good catch though.